critiquebrainz/frontend/external/musicbrainz_db/artist.py

```diff
@@ -15,9 +16,17 @@ def get_artist_by_id(mbid):
     key = cache.gen_key(mbid)
     artist = cache.get(key)
     if not artist:
-        artist = fetch_multiple_artists(
+        multiple_artists = fetch_multiple_artists(
```

Comment:
One thing that bothers me is the name `multiple_artists` or `multiple_*` since we pass only a single mbid, I'd recommend  having a function `fetch_multiple_artists` (critiquebrainz version), that in turns calls the bu version and returns unknown if not found. (Similar for others)

Reply:
That would avoid code duplication, for example, in `entities.py`

Reply:
Updated code and fixed a bug I found in utils.py